cognite/neat/constants.py

```diff
@@ -9,5 +9,6 @@
 EXAMPLE_WORKFLOWS = EXAMPLES_DIRECTORY / "workflows"
 
 TNT_TRANSFORMATION_RULES = EXAMPLE_RULES / "Rules-Nordic44-to-TNT.xlsx"
+SIMPLE_TRANSFORMATION_RULES = EXAMPLE_RULES / "sheet2cdf-transformation-rules.xlsx"
```

Comment:
File is referenced but not added to examples folder

Reply:
https://github.com/cognitedata/neat/blob/main/cognite/neat/examples/rules/sheet2cdf-transformation-rules.xlsx

Reply:
ok , so it's one of existing files and there are not changes in content or format of the file ?

Reply:
Indeed, no changes.